clang/include/clang/Driver/Options.td

```diff
@@ -5392,7 +5392,9 @@ def regcall4 : Flag<["-"], "regcall4">, Group<m_Group>,
   MarshallingInfoFlag<LangOpts<"RegCall4">>;
 def save_temps_EQ : Joined<["-", "--"], "save-temps=">, Flags<[NoXarchOption]>,
   Visibility<[ClangOption, CC1Option, FlangOption, FC1Option]>,
-  HelpText<"Save intermediate compilation results.">;
+  HelpText<"Save intermediate compilation results. The <arg> option can be set "
```

Comment:
`<arg>` is an argument, not an option, but it is unnecessary to mention it is an argument.

`<arg> can be set to 'cwd' for ..., or 'obj' ...`

Reply:
The committed version does not contain single quotes. I'll add them.

Reply:
Ah good catch, thanks